com.ibm.wala.shrike/src/com/ibm/wala/shrikeCT/BootstrapMethodsReader.java

```diff
@@ -146,17 +146,12 @@ public Object callArgument(ClassLoader cl, int i) {
                   default:
                     assert false : "invalid type " + t;
                 }
-              } catch (IllegalArgumentException e) {
-                assert false : e;
-              } catch (InvalidClassFileException e) {
-                assert false : e;
-              } catch (ClassNotFoundException e) {
-                assert false : e;
-              } catch (NoSuchMethodException e) {
-                assert false : e;
-              } catch (SecurityException e) {
-                assert false : e;
-              } catch (IllegalAccessException e) {
+              } catch (IllegalArgumentException
```

Comment:
I don't like the use of `assert` here.  Can we re-throw instead, wrapping in a `RuntimeException`?

Reply:
Done.